dymos/transcriptions/pseudospectral/components/state_interp_comp.py

```diff
@@ -242,7 +241,8 @@ def _compute_partials_radau(self, inputs, partials):
 
             dstau_dt_x_size = np.repeat(dstau_dt, size)[:, np.newaxis]
 
-            partials[xdotc_name, xd_name] = self.jacs['Ad'][name].multiply(dstau_dt_x_size).data
+            dxdotc_dxd = self.jacs['Ad'][name].multiply(dstau_dt_x_size)
+            partials[xdotc_name, xd_name] = dxdotc_dxd.data
```

Comment:
Is it ok to still use data here? Will it work the same way in old and new scipy?

Reply:
It seems to work fine, ultimately the issue was a change in formats to the matrices involved that I've resolved